src/test/java/com/xceptance/neodymium/testclasses/webDriver/ValidateWebDriverMaxReuseWithTwoWebDriver.java

```diff
@@ -0,0 +1,268 @@
+package com.xceptance.neodymium.testclasses.webDriver;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.aeonbits.owner.ConfigFactory;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import com.browserup.bup.BrowserUpProxy;
+import com.xceptance.neodymium.NeodymiumRunner;
+import com.xceptance.neodymium.module.statement.browser.multibrowser.Browser;
+import com.xceptance.neodymium.module.statement.browser.multibrowser.WebDriverCache;
+import com.xceptance.neodymium.module.statement.browser.multibrowser.WebDriverStateContainer;
+import com.xceptance.neodymium.tests.NeodymiumTest;
+import com.xceptance.neodymium.tests.NeodymiumWebDriverTest;
+import com.xceptance.neodymium.util.Neodymium;
+
+@RunWith(NeodymiumRunner.class)
+public class ValidateWebDriverMaxReuseWithTwoWebDriver
```

Comment:
- [ ] The purpose of this test class is not entirely clear. Maybe a comment would help.
- [ ] Typo. It's WebDriver**s**

Reply:
done